### ISSUE TITLE
Accessibility: [High Contrast]: The text cannot be shown when focus "Close" button of ToolStrip item in Print Preview dialog

### ISSUE THREAD

**Issue by Jessie-Zhang01** [dotnet/winforms#1421]
* .NET Core Version: 3.0.100 Preview8 13316 from master branch
* Have you experienced this same bug with .NET Framework?: Yes.

**Repro Steps:**

1. Open control panel and select the respective high contrast theme.
2. Open the attached application [PrintDemo .zip](https://github.com/dotnet/winforms/files/3405539/PrintDemo.zip), build and run. 
3. Click the 'Preview' button to show the Print preview dialog then focus on ‘Close’ button.

**Actual:**
The text cannot be shown when focus "Close" button of 'Print Preview' dialog on the Black and White theme. See following screenshot for Black and White theme.
![image](https://user-images.githubusercontent.com/45864649/61443867-e71d3280-a97c-11e9-9dca-ba72ff67e889.png)

**Expected:**
The text should be shown when focus "Close" button of 'Print Preview' dialog on the Black and White theme. See following screenshot for # 1 and # 2 theme.
![image](https://user-images.githubusercontent.com/45864649/61444150-6a3e8880-a97d-11e9-88b0-23d83b6f178f.png)

**Comment by merriemcgaw** [comment]
@M-Lipin or @vladimir-krestov can you guys take a look at this?

**Comment by Marie-Shi** [comment]
Verified this bug with .Net core 3.0.100-rc1-014190 from release branch, the text can be shown when focus "Close" button of 'Print Preview' dialog on the Black and White theme. Please see the following screenshots:
![HC-Black](https://user-images.githubusercontent.com/45864017/64593440-ad96ff80-d3e0-11e9-99c9-8a7516fe698f.png)
![HC-White](https://user-images.githubusercontent.com/45864017/64593463-b7206780-d3e0-11e9-8ba9-8d287ccc4db3.png)
